index.js

```diff
@@ -1,46 +1,51 @@
-
-module.exports = BroccoliMergeTrees;
-BroccoliMergeTrees.prototype = Object.create(Plugin.prototype);
-BroccoliMergeTrees.prototype.constructor = BroccoliMergeTrees;
-function BroccoliMergeTrees(inputNodes, options) {
-  if (!(this instanceof BroccoliMergeTrees)) return new BroccoliMergeTrees(inputNodes, options);
-  options = options || {};
-  var name = 'broccoli-merge-trees:' + (options.annotation || '');
-  if (!Array.isArray(inputNodes)) {
-    throw new TypeError(name + ': Expected array, got: [' + inputNodes +']');
-  }
-  Plugin.call(this, inputNodes, {
-    persistentOutput: true,
-    needsCache: false,
-    annotation: options.annotation
-  });
-  this.inputNodes = inputNodes;
-  this.options = options;
-}
+const Plugin = require('broccoli-plugin');
+const MergeTrees = require('merge-trees');
 
-BroccoliMergeTrees.prototype.build = function() {
-  if (this.mergeTrees == null) {
-    // Defer instantiation until the first build because we only
-    // have this.inputPaths and this.outputPath once we build.
-    this.mergeTrees = new MergeTrees(this.inputPaths, this.outputPath, {
-      overwrite: this.options.overwrite,
-      annotation: this.options.annotation
+class BroccoliMergeTrees extends Plugin{
+  constructor (inputNodes, options = {}) {
+    if (!Array.isArray(inputNodes)) {
+      throw new TypeError(name + ': Expected array, got: [' + inputNodes +']');
+    }
+    super(inputNodes, {
+      persistentOutput: true,
+      needsCache: false,
+      annotation: options.annotation
     });
+
+    let name = 'broccoli-merge-trees:' + (options.annotation || '');
+
+    this.inputNodes = inputNodes;
+    this.options = options;
   }
 
-  try {
-    this.mergeTrees.merge();
-  } catch(err) {
-    if (err !== null && typeof err === 'object') {
-      let nodesList = this.inputNodes.map((node, i) => `  ${i+1}.  ${node.toString()}`).join('\n');
-      let message = `${this.toString()} error while merging the following:\n${nodesList}`;
+  build() {
+    if (this.mergeTrees == null) {
+      // Defer instantiation until the first build because we only
+      // have this.inputPaths and this.outputPath once we build.
+      this.mergeTrees = new MergeTrees(this.inputPaths, this.outputPath, {
+        overwrite: this.options.overwrite,
+        annotation: this.options.annotation
+      });
+    }
+
+    try {
+      this.mergeTrees.merge();
+    } catch(err) {
+      if (err !== null && typeof err === 'object') {
+        let nodesList = this.inputNodes.map((node, i) => `  ${i+1}.  ${node.toString()}`).join('\n');
+        let message = `${this.toString()} error while merging the following:\n${nodesList}`;
 
-      err.message = `${message}\n${err.message}`;
+        err.message = `${message}\n${err.message}`;
+      }
+      throw err;
     }
-    throw err;
   }
+}
+
+module.exports = function mergeTrees(...params) {
+  return new BroccoliMergeTrees(...params);
 };
+
+module.exports.BroccoliMergeTrees = BroccoliMergeTrees;
```

Comment:
Let's call it `MergeTrees`:

```suggestion
module.exports.MergeTrees = MergeTrees;
```

This way folks that do want the class usage can do:

```js
const { MergeTrees } = require('broccoli-merge-trees');
```